components/community-groups/CommunityGroupsCard.jsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Card, CardContent, Typography, Button, InputLabel, 
+  TextField, FormControl
+} from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import NativeSelect from '@material-ui/core/NativeSelect';
+// EditableCG here
+// import EditableEvent from '../editableEvent/EditableEvent';
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 250,
+  },
+  deleteButton: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    color: '#ffffff',
+    backgroundColor: '#dd7d1b',
+  },
+  editButton: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    color: '#ffffff',
+    backgroundColor: '#007398',
+  },
+  buttonGroup: {
+    alignItems: 'center'
+  },
+  cardControl: {
+    width: '18vw',
+    height: '20vw',
+    minWidth: 275,
+  },
+  formControl: {
+    display: 'flex',
+    flexDirection: 'row',
+  }
+});
+
+// generate String to display group of leaders
+const generateStringOfGroup = (group) => {

```

Comment:
If this was a generic utility being used in multiple places I would be fine with a name like this. Since it is only being used for leader names it should probably have a more specific name.

Reply:
Makes sense, I wanted this to be a more generic utility for all string arrays, but currently only used by leader names